clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/deploy/DescriptionValidator.groovy

```diff
@@ -25,4 +32,39 @@ public abstract class DescriptionValidator<T> {
   }
 
   abstract void validate(List priorDescriptions, T description, Errors errors)
+
+  @Autowired(required = false)
+  FiatPermissionEvaluator permissionEvaluator
+
+  void authorize(T description, Errors errors) {
+    if (!permissionEvaluator) {
+      return
+    }
+
+    Authentication auth = SecurityContextHolder.context.authentication
+
+    if (description instanceof ApplicationNameable) {
+      ApplicationNameable asApp = description as ApplicationNameable
```

Comment:
If that is the norm then that's fine ... Even in this method alone 'WRITE' is repeated 3 times.  I would personally DRY that up in a constant.